lib/src/10_auth/auth_screen.dart

```diff
@@ -81,6 +85,20 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
     return widget.phrases[phrase] ?? message;
   }
 
+  @override
+  void dispose() {
+    _emailNode.dispose();
+    _passwordNode.dispose();
+    super.dispose();
+  }
+
+  @override
```

Comment:
What's the goal of this override?

Reply:
In https://flutter.dev/docs/cookbook/forms/focus , I found:

@override
  void dispose() {
    // Clean up the focus node when the Form is disposed.
    myFocusNode.dispose();

    super.dispose();
  }

Reply:
I mean setState:

void setState(fn) {
    if (mounted) {
      super.setState(fn);
    }
  }

Reply:
I was getting runtime exception which looked something like:
setState() called after dispose().
This error happens if you call setState() on a State object for a widget that no longer appears in the widget tree (e.g., whose parent widget no longer includes the widget in its build). This error can occur when code calls setState() from a timer or an animation callback. The preferred solution is to cancel the timer or stop listening to the animation in the dispose of the () callback. Another solution is to check the "mounted" property of this object before calling setState() to ensure the object is still in the tree.
This error might indicate a memory leak if setState() is being called because another object is retaining a reference to this State object after it has been removed from the tree. To avoid memory leaks, consider breaking the reference to this object during dispose of ().
I checked online for this exception and thought it could be due to submit() returning a Future in handleKey. 
Checking for mounted was stated as the solution and this piece of code took care of the runtime exception.